### DISCUSSION TITLE
图标题caption和插图索引标题

### DISCUSSION THREAD

**Discussion by sdfgh468** [sjtug/SJTUThesis#985] [提问] [answered]
求问，如何在博士学位论文模板中的图\caption中增加对图的解释，如下图所示，
![image](https://github.com/sjtug/SJTUThesis/assets/22966522/1a88a4a1-8122-4734-8084-7747b30386c7)

但是在论文前几页的插图索引部分，只显示该图标题的第一句话
![image](https://github.com/sjtug/SJTUThesis/assets/22966522/5137ad23-7344-43c6-ae72-02eb7e00ee66)

**Comment by LogCreative** [comment]
请提供对应的代码。有可能使用了 `\caption[短标题]{长标题}` 的语法。

**Reply by sdfgh468** [reply]
上面截图是网上下的别人论文，我没有实现这个的代码，我的困惑在于我想在图的标题部分写一些解释的文字，有点长，就像我上面截图里那样五行左右，如果就是用模板里的\bicaption{中文}{英文}，会导致插图索引部分的图标题特别长，不美观，我只想在插图索引部分显示caption中的第一句话，求问怎么做呢

**Reply by sdfgh468** [reply]
我查网上，确实有你说的\caption[短标题]{长标题}，但是学位论文中一般是中英双语标题\bicaption{中文}{英文}，如果我想要这两个功能合并，就是短长标题，同时双语，怎么做呢

**Reply by LogCreative** [reply, marked as answer]
```latex
\bicaption[中文短标题]{中文长标题}[英文短标题]{英文长标题}
```
